skymp5-server/cpp/server_guest_lib/WorldState.h

```diff
@@ -154,6 +154,7 @@ class WorldState
                      std::chrono::system_clock::duration dur);
   std::optional<std::chrono::system_clock::duration> GetRelootTime(
     std::string recordType) const;
+  void AddLoadOrderElement(std::string fileName);
```

Comment:
this is not needed. you can use the existing variable espmFiles.

index in this vector == first 2 digits of form id